tests/support_server/setup.pm

```diff
@@ -130,6 +130,9 @@ sub setup_networks {
 
     $setup_script .= "FIXED_NIC=`grep $net_conf->{fixed}->{mac} /sys/class/net/*/address |cut -d / -f 5`\n";
     $setup_script .= "iptables -F\n";
+    $setup_script .= "iptables -A INPUT -i \$FIXED_NIC -p tcp -m tcp  -j ACCEPT\n";
+    $setup_script .= "iptables -A INPUT -i \$FIXED_NIC -p udp -m udp  -j ACCEPT\n";
+    $setup_script .= "iptables -A OUTPUT -m state --state ESTABLISHED,RELATED -j ACCEPT\n";
```

Comment:
TBH, I would allow all traffic on that interface:
```
iptables -A INPUT -i \$FIXED_NIC -j ACCEPT
iptables -A OUTPUT -i \$FIXED_NIC -j ACCEPT
```

Or then configure it to allow VNC and SSH, but we are not going to test firewall here and we need it only because of NAT.

Reply:
VR http://aquarius.suse.cz/tests/3132 and http://aquarius.suse.cz/tests/3129